gateway/managers/syncman/syncman_runner.go

```diff
@@ -288,6 +291,107 @@ func (s *Manager) HandleRunnerGetServiceRoutingRequest(admin *admin.Manager) htt
 	}
 }
 
+// HandleRunnerGetServiceLogs handles requests of the runner
+func (s *Manager) HandleRunnerGetServiceLogs(admin *admin.Manager) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		userToken := utils.GetTokenFromHeader(r)
+
+		vars := mux.Vars(r)
+		projectID := vars["project"]
+		utils.LogDebug("Forwarding request to runner for getting service logs", "syncman", "HandleRunnerGetServiceLogs", map[string]interface{}{})
+
+		_, err := admin.IsTokenValid(userToken, "service-logs", "read", map[string]string{"project": projectID})
+		if err != nil {
+			logrus.Errorf("error handling forwarding runner request failed to validate token -%v", err)
+			_ = utils.SendErrorResponse(w, http.StatusUnauthorized, err.Error())
+			return
+		}
+
+		// Create a context of execution
+		_, cancel := context.WithTimeout(r.Context(), 60*time.Second)
+		defer cancel()
+
+		// http: Request.RequestURI can't be set in client requests.
+		// http://golang.org/src/pkg/net/http/client.go
+		r.RequestURI = ""
+
+		// Get host from addr
+		host := strings.Split(s.runnerAddr, ":")[0]
+
+		// Change the request with the destination host, port and url
+		r.Host = host
+		r.URL.Host = s.runnerAddr
+
+		// Set the url scheme to http
+		r.URL.Scheme = "http"
+
+		token, err := admin.GetInternalAccessToken()
+		if err != nil {
+			logrus.Errorf("error handling forwarding runner request failed to generate internal access token -%v", err)
+			_ = utils.SendErrorResponse(w, http.StatusInternalServerError, err.Error())
+			return
+		}
+		r.Header.Set("Authorization", fmt.Sprintf("Bearer %s", token))
+
+		// TODO: Use http2 client if that was the incoming request protocol
+		response, err := http.DefaultClient.Do(r)
+		if err != nil {
+			_ = utils.SendErrorResponse(w, http.StatusInternalServerError, err.Error())
+			return
+		}
+		defer utils.CloseTheCloser(response.Body)
+
+		streamData := false
+		// Copy headers and status code
+		for k, v := range response.Header {
+			// check if data is available in chunks
+			if k == "X-Content-Type-Options" && v[0] == "nosniff" {
+				streamData = true
+			}
+			w.Header().Set(k, v[0])
+		}
+
+		if streamData {
```

Comment:
You dont seem to send any response if streamData is false

Reply:
should i send error response specifying you need to set this header

Reply:
Ok. Thats works